pkg/virt-handler/controller.go

```diff
@@ -196,11 +196,11 @@ func (c *BaseController) isMigrationSource(vmi *v1.VirtualMachineInstance) bool
 		!vmi.Status.MigrationState.Completed
 }
 
-func (c *BaseController) claimDeviceOwnership(virtLauncherRootMount *safepath.Path, deviceName string) error {
+func (c *BaseController) tryClaimDeviceOwnership(virtLauncherRootMount *safepath.Path, deviceName string, ignoreErrorWithEmulation bool) error {
```

Comment:
Perhaps it's just me, but "ignoreErrorWithEmulation" is slightly confusing, "ignoreEmulationError" is a tad clearer. Alternatively, would "isHypervisorDevice" be a valid name for the parameter?

Reply:
Thanks for the suggestion. Absolutely, `isHypervisorDevice` sounds like a better fit.